spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1570,4 +1570,19 @@ describe Crystal::Formatter do
     end
     ex.line_number.should eq(5)
   end
+
+  # #8197
+  assert_format <<-CODE
+    foo
+      .foo1(bar
+        .bar1
+        .bar2)
```

Comment:
I think this format should be preserved.

For example if I write this:

```crystal
foo
  .foo1(bar
    .bar1
    .bar2)
  .foo2(baz
    .baz2
    .baz3)
```

I'd like it to be formatted like that (without change).

Reply:
This spec intend to what you say. What is problem?

Reply:
Oooooh... sorry! For some reason I thought the lines below these ones are the formatted version, but now I see they are separate test cases. 😊